todo_app/models.py

```diff
@@ -0,0 +1,28 @@
+from pytest import Item
+
+from todo_app.enums import Status
+
+
+class ViewModel:
+    def __init__(self, items: list[Item]):
+        self._items = items
+ 
+    @property
+    def items(self):
+        return self._items
+    
+    @property
+    def incomplete_items(self):
+        incomplete_items: list[Item] = []
+        for item in self._items:
+            if item.status == Status.INCOMPLETE:
+                incomplete_items.append(item)
+        return incomplete_items
```

Comment:
Nice and clean👍 
Not necessarily a recommendation as it depends on preference for readability, but I always think interesting to see the list-comprehension approach for a more concise solution, something like:
```python
    @property
    def incomplete_items(self):
        return [item for item in self._items if item.status == Status.INCOMPLETE]
```